ntbs-service/Pages/Notifications/Denotify.cshtml.cs

```diff
@@ -42,14 +42,19 @@ public async Task<IActionResult> OnGetAsync(int id)
                 return NotFound();
             }
 
+            await AuthorizeAndSetBannerAsync();
+            if (!HasEditPermission)
+            {
+                // Should not be able to get onto the denotify page for a notification user does not have edit access to
```

Comment:
This is just supposed to be a failsafe as we won't show the button - however I think I forgot to actually hide it when HasEditPermission is false! TBD